applications/longshot/main.tf

```diff
@@ -44,30 +44,11 @@ module "app" {
   with_newrelic          = false
 }
 
-module "database" {
-  source = "../../components/mariadb_instance"
-
-  deprecated = var.deprecated
-
-  name              = var.name
-  environment       = var.environment
-  database_name     = "longshot"
-  instance_class    = var.environment == "production" ? "db.t2.medium" : "db.t2.micro"
-  allocated_storage = var.environment == "production" ? 100 : 5
-
-  subnet_group    = "default-vpc-7899331d"
-  security_groups = ["sg-c9a37db2"]
-}
-
-module "iam_user" {
-  source = "../../components/iam_app_user"
-  name   = var.name
-}
-
 module "storage" {
-  source  = "../../components/s3_bucket"
-  name    = var.name
-  private = true
+  source   = "../../components/s3_bucket"
+  name     = var.name
+  private  = true
+  archived = true
```

Comment:
Since this is applied with a lifecycle policy, objects will be moved into Glacier after 30 days.